warehouse/forklift/legacy.py

```diff
@@ -46,6 +50,17 @@
 
 PATH_HASHER = "blake2_256"
 
+ALL_STDLIB = set(
+    chain.from_iterable(
+        [stdlib_list.stdlib_list(version)
```

Comment:
This would be slightly better as ``(...)`` instead of ``[...]`` to make this is a generator expression. That will prevent us from creating an intermediate list.